librepo/fastestmirror.c

```diff
@@ -39,6 +39,11 @@
 #define CACHE_GROUP_METADATA    ":_librepo_:"   // Group with metadata
 #define CACHE_KEY_TS            "ts"            // Timestamp
 #define CACHE_KEY_CONNECTTIME   "connectime"    // Time of response
+                                                // FIXME: next time the version
+                                                //        is updated, please
+                                                //        fix this typo! It
+                                                //        should obviously have
+                                                //        been "connecttime".
```

Comment:
I'm thinking we should probably add a short reference to where the `SOVERSION` is defined in `librepo/CMakeLists.txt`, because otherwise this could be easily missed.

Reply:
I've been thinking about a way to make this more prominent, but I'm afraid that the cache structure version isn't actually coupled to the SOVERSION. The former has never been bumped up either so far.

I guess incrementing it with the next SOVERSION bump won't hurt, though, so why not.

Reply:
Ah, sorry, I wasn't paying enough attention, I thought it's on the binary interface, but it is not, it is stored in the cache file. Unrelated to `SOVERSION`...

So yeah... looking at the code it probably wouldn't even break anything if we just changed it, as the cache would  simply be ignored (and I assume rewritten with the new key). But it's probably not enough reason to do it to fix the typo. (@j-mracek, @jrohel thoughts?)
If we won't be fixing it, could I ask to put the FIXME comment on the line above the constant and keep it a single line to make it a bit more subtle? :slightly_smiling_face:

Thanks!

Reply:
Right. I mean, I don't care. Updating the cache version means literally changing one digit. I just don't think that it makes a lot of sense. Also, it's just a stupid typo no user is likely to see, so we don't have to over-engineer stuff here.

Updated the comment/moved it above.